File: algorithms/q8/solution.py
def linearSearch(arr,target_int):
    for i in range(len(arr)-1):
        if(arr[i] == target_int):
            return True
    return False

arr = [2,34,35,1,4,5,3,22]
x = int(input("Enter the target intger: "))

print(linearSearch(arr,x))

File: algorithms/q8/solution.cpp
#include<iostream>
#include<vector>
using namespace std;

bool linearSearch(vector<int> arr,int x){
    for(int i = 0;i < arr.size();i++){
        if(arr[i] == x){
            return true;
        }
    }
    return false;
}

int main(){
    vector<int> arr = {2,3,4,1,4,1,44,5,3,2};
    cout << linearSearch(arr, 3) << endl;

    return 0;
}